### DISCUSSION TITLE
pls help me in my opensource journey

### DISCUSSION THREAD

**Discussion by kushalShukla-web** [cncf/mentoring#718] [Q&A] [answered]
hey guys im from tier -3 college and know  some web-d . but im quite bore with that job and want to learn something new . so im going to start with cloud services. but dont know where to start and what to do .  and also previously i done very few opensource contribution . can any body pls help me through this and show me some projects so that   i too improve this organization with my knowledge

**Comment by nate-double-u** [comment, marked as answer]
Hi @kushalShukla-web!

If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contributors](https://contribute.cncf.io/) page.

If you're interested in being a mentee, you can also take a look around this repo and see what opportunities there are. Some projects listed here have issues marked "Good first issue" which you can take a look at. It's good to communicate and work with projects that you're interested in doing a mentorship with.
